crates/red_knot_python_semantic/src/types/infer.rs

```diff
@@ -1827,15 +1826,17 @@ impl<'db> TypeInferenceBuilder<'db> {
     /// Look up a name reference that isn't bound in the local scope.
     fn lookup_name(&self, name: &ast::name::Name) -> Type<'db> {
         let file_scope_id = self.scope.file_scope_id(self.db);
-        let symbols = self.index.symbol_table(file_scope_id);
-        let symbol = symbols
+        let is_defined = self
+            .index
+            .symbol_table(file_scope_id)
             .symbol_by_name(name)
-            .expect("Expected the symbol table to create a symbol for every Name node");
+            .expect("Expected the symbol table to create a symbol for every Name node")
```

Comment:
TIL I learned from @dhruvmanila. Sharing for you https://doc.rust-lang.org/std/error/index.html#common-message-styles